**data/748 LTTR representative proteins** <br>
<i>748_clusters.zip<i> Resulting CD Hit clustering of the 86k sequences to the 748 representative LTTR protein sequences. <br>
<i>748_seqs.zip<i> Resulting newick of the 748 representative LTTR protein sequences after using the Phyml Tool. <br>
<i>86k_seqs.zip<i> Resulting 86k LTTR protein sequences (fasta). <br>
<i>muscle_alignment.zip<i> Resulting muscle alignment of the 748 representative LTTR protein sequences. <br>

**data/BLAST_and_DNA_seqs** <br>
<i>BLAST_screenshots_most_variable.zip<i> Screenshots of the BLAST results of the most variable LTTRs. <br>
<i>Blast results.xlsx<i> Resulting BLAST results of the most variable & least variable LTTRs in excel format. <br>
<i>all_dna_sequences.zip<i> Resulting 86k LTTR DNA sequences (fasta). <br>

**data/Genomes_and_isolates_data** <br>
<i>ITOL Scripts-20220518T094541Z-001.zip<i> Scripts used to create trees in iTol. <br>
<i>MUSCLE_FILES-20220518T094547Z-001.zip<i> Resulting muscle alignment of the most variable LTTR clusters. <br>
<i>NEWICK_FILES-20220518T094549Z-001.zip<i> Resulting newick of the most variable clusters. <br>
<i>Trees-20220518T094555Z-001.zip<i> PDF versions of all trees with genome data created of the most variable clusters. <br>
<i>geneids_and_genomes.csv<i> LTTR and corresponding genome. <br>
<i>isolates.xlsx<i> Information on the 699 genomes. <br>
<i>phyml_parameters_screeshot.JPG<i> Screenshot showing parameters used when using Phyml online tool. <br>

**data/PqsR** <br>
<i>Metadata Tree PqsR.pdf<i> PqsR metadata tree in PDF form. <br>
<i>all_pqsr_80<i> PqsR protein sequences. <br>
<i>pqsr_muscle<i> Resulting muscle alignment of the PqsR cluster. <br>
<i>pqsr_phyml_tree.nwk<i> PqsR newick file. <br>
<i>pqsr_seqs_80.clstr<i> Resulting CD Hit clustering of PqsR at 80%. <br>

**data/Variability** <br>
<i>80__top_and_lowest_ten_variable.R<i> Rscript used to create pie charts. <br>
<i>LEAST VAR.png<i> Piecharts of least variable LTTRs. <br>
<i>MOST  VAR.png<i> Piecharts of most variable LTTRs. <br>
<i>Ten Least Variable-20220518T101839Z-001.zip<i> Excel files showing number of clusters and number of sequences in each cluster of each of the least variable LTTRs. <br>
<i>Ten Most Variable-20220518T101837Z-001.zip<i> Excel files showing number of clusters and number of sequences in each cluster of each of the most variable LTTRs. <br>
<i>clade infor for 10 top and least variable.xlsx<i> File showing which clade each of the LTTRs can be found. <br>
<i>lowest_ten_80.xlsx<i> Variance values for least variable LTTRs. <br>
<i>most_variable.xlsx<i> Variance values for most variable LTTRs. <br>
<i>variability_within_clusters_new..xlsx<i> Variance values for all clusters when CD Hit performed at 80%. <br>

**data/clade_information** <br>
<i>CLADE TABLE.xlsx<i> Summarised nformation of the 9 clades - total number of LTTRs, characterised members, number of homologues etc. <br>
<i>clade_info.xlsx<i> Raw information of LTTRs found in each of the clades. <br>

**data/mkt** <br>
<i>outlier_dna_seqs<i> DNA sequences used as outliers for test. <br>
<i>McDonald Kreitman Results.xlsx<i> Results in excel format. <br>
<i>MkT screenshot.jpg<i> Screenshot showing parameters used when using MkT online tool. <br>
<i>dna_seqs-20220518T093621Z-001.zip<i> LTTR DNA sequences used for test. <br>
